libres_util/include/ert/res_util/res_log.h

```diff
@@ -28,6 +28,16 @@ extern "C" {
 
 #include <ert/util/log.h>
 
+//Same as pythons default log levels.
+typedef enum {
+  CRITICAL=0, //OOM.
```

Comment:
Maybe these should be in the `libecl/log` implementation; and prefixed with `LOG_`- i.e. `LOG_ERROR` and `LOG_DEBUG`?